include/fizzy/fizzy.h

```diff
@@ -406,14 +417,13 @@ FizzyInstance* fizzy_instantiate(const FizzyModule* module,
 /// FizzyImportedFunction::name strings.
 ///
 /// @note
-/// Function expects @p imported_globals to be in the order of imports defined in the module.
-/// No validation is done on the number of globals passed in, nor on their order.
-/// When number of passed globals or their order is different from the one defined by the
-/// module, behaviour is undefined.
+/// Globals in @a imported_globals are allowed to be in any order and allowed to include some
```

Comment:
Are we using `#imported_globals` now? Or not for arguments?

Reply:
Not for arguments, I think I tried it already https://github.com/wasmx/fizzy/pull/681#issuecomment-762415754